tests/common/osutil/test_default.py

```diff
@@ -777,7 +781,7 @@ def test_enable_firewall_should_not_set_firewall_if_the_drop_rule_exists(self):
             success = osutil.DefaultOSUtil().enable_firewall(dst_ip=mock_iptables.destination, uid=mock_iptables.uid)
 
             self.assertTrue(success, "Enabling the firewall was not successful")
-            self.assertEqual(len(mock_iptables.command_calls), 1, "Incorrect number of calls to iptables: [{0}]". format(mock_iptables.command_calls))
+            self.assertEqual(len(mock_iptables.command_calls), 2, "Incorrect number of calls to iptables: [{0}]". format(mock_iptables.command_calls))
```

Comment:
Can you please also add a testcase for the new scenario that you've introduced, where you check if the TCP rule exists or not and add just that one.

Reply:
Added few test cases under test_update.py